docs/production-deployment/cloud/nexus/index.mdx

```diff
@@ -18,71 +18,368 @@ keywords:
+
+In Temporal Cloud, the Nexus Machinery routes Nexus requests across Namespace boundaries through a secure mTLS Envoy mesh, where they are sync matched to a Worker in the handler Namespace, which processes the Nexus request.
+
+Nexus caller Workflows will send commands to Temporal Cloud to schedule or cancel a Nexus Operation in the caller Namespace.
+Handler Workers will poll for Nexus tasks in the handler Namespace.
+
+Nexus Operation handlers can be implemented as [Asynchronous Operations](/nexus/services-operations#asynchronous-operation-lifecycle) or [Synchronous Operations](/nexus/services-operations#asynchronous-operation-lifecycle), using Temporal SDK builders that enable running a Nexus Service in a Worker, typically the same Worker as the underlying primitives the Nexus Service abstracts.
+
+For example, an asynchronous operation that starts a Workflow in a different Namespace:
+
+Unlike a traditional RPC, an arbitrary-duration Nexus Operation has an identity (ID) that lasts beyond the process lifetimes of the caller and handler, and can be used to re-attach to a long-lived Nexus Operation, for example one backed by a Temporal Workflow.
+
+When you schedule a Nexus Operation in a caller Workflow, a command is sent to Temporal to schedule the Operation, and then the Temporal Nexus Machinery is responsible for making the Nexus RPC calls on your behalf.
+This means you don’t have to use Nexus RPC directly, only the Temporal SDK along with a Temporal Service.
+
+![NexusYourCloud](/img/nexus/NexusYourCloud.png)
+
+Similar to executing an Activity, to execute a Nexus Operation from a Workflow:
+
+1. Caller Workflow will schedule a Nexus Operation using the Temporal SDK, which sends a `ScheduleNexusOperation` command in the existing `RespondWorkflowTaskCompleted` RPC which is recorded as `NexusOperationScheduled` in the caller’s Workflow history.
+2. Nexus handler Worker uses the Temporal SDK to `PollNexusTaskQueue` and process the Nexus request.
+3. Nexus handler Worker sends a `RespondNexusTaskCompleted` through the SDK, with the result of the Nexus request, which may be one of: Started, Completed, Failed, TimedOut, or Canceled. See [Nexus Operation Events](/references/events) for more info
+4. Caller Workflow then processes the Nexus Operation event through `PollWorkflowTaskQueue`.
+
+See the [Temporal Nexus](/nexus) encyclopedia entry for more details.
+
+## Access Control
+
+Like Namespaces, a Nexus Endpoint is an account-scoped resource that is global within a Temporal Cloud account.
+Any Developer role (or higher) in an account, who is also a Namespace Admin on the endpoint’s target Namespace, can manage (create, update, or delete) a Nexus Endpoint.
+All users with a Read-only role (or higher) in an account, can view and browse the full list of Endpoints.
+
+Runtime access from a Workflow in a caller Namespace to a Nexus Endpoint is controlled by an allowlist policy (of caller Namespaces) for each Endpoint in the Nexus API registry.
```

Comment:
I think whitelist might be more globally understood than allowist policy. I think this can be rephrased to be stronger.

Also a general note, there are a lot of parentheses in these pull requests. "Runtime access to Nexus Endpoints from Workflows is controlled by a whitelist of originating Namespaces." or something like that.

Reply:
Also, I have a meeting and have to stop here. I hope the feedback is constructive and helpful. Please let me know if I can improve the way I provide feedback, as I'm always looking to become better at this.

Reply:
Much appreciated!